CONTRIBUTING.md

```diff
@@ -1,7 +1,72 @@
+# Thank you for considering contributing!
+
+Contributions are always welcome. We accept [bug fixes](#bugs),
+[proposals and new ideas](#proposals-for-new-content), or simply [editorial
+improvements](#editorial-improvements).
+
+By submitting a pull request you are implicitly adhering to the
+[contribution agreement](#contribution-agreement)
+
+## Bugs
+
+You can report a bug by creating an
+[issue](https://github.com/ARM-software/acle/issues). You can also fix
+it yourself with a [pull
+request](https://github.com/ARM-software/acle/pulls).
+
+## Proposals for new content
+
+We are open to any proposal, including the introduction of new sets of
+intrinsics.
+
+For adding new intrinsics or preprocessor macros, we recommend
+presenting them in a [pull
+request](https://github.com/ARM-software/acle/pulls), together with
+the following helpful information:
+
+1. A rationale in favour of these intrinsics or macros.
+2. A design document to keep track of the reasoning that made the
+   proposal reach its current state.
+
+Please note that this extra information is not a requirement for
+submitting new content. Contributors are trusted to use their judgment
+in deciding whether or not the proposal needs this information.
+Providing the information is recommended to make it easier
+for new ideas to be discussed and possibly accepted, especially for
+changes of great impact.
```

Comment:
Arm recommends that you add this information so that it is easier for new ideas to be discussed...